src/services/Staffing.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import moment from 'moment';
+
+/**
+ * Generate a list of 20 weeks, from four weeks ago to 16 weeks in future.
+ *
+ * @return Array<moment>
+ */
+export function createWeeks(): Array<moment> {
+  const weeks = [];
+  for (let i = -4; i <= 16; i += 1) {
```

Comment:
i'd suggest if you use lodash to do a range(20).map(index => ....) I find it more functional :)